wrappers/apexgateway/thundra_apex_gateway.go

```diff
@@ -0,0 +1,34 @@
+package apexgateway
+
+import (
+	"context"
+	"github.com/apex/gateway"
+	"github.com/thundra-io/thundra-lambda-agent-go/v2/thundra"
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+func ListenAndServe(h http.Handler) error {
```

Comment:
what about another `ListenAndServe` and `wrapper` for API Gateway v2 (https://github.com/apex/gateway/blob/master/v2/gateway.go)?

Reply:
package apexgatewayv2 added for this requirement